.github/workflows/build-pr.yml

```diff
@@ -65,7 +65,10 @@ jobs:
             [Pull request has been updated, please wait for latest changes to be pushed to npm...]
 
       - name: Install dependencies
-        run: npm install
+        run: |
+          npm install
+          cd cli
+          npm install
```

Comment:
Even if we don't publish the package yet, I'd add it to ``lerna.config`` and then there's no need for this, as it will bootstrap the dependencies